ipaclient/remote_plugins/schema.py

```diff
@@ -64,6 +64,13 @@
 }
 
 
+class BytesEncoder(json.JSONEncoder):
+    def default(self, obj):
```

Comment:
@tiran is there something undesirable about using the class (apart from a tiny bit more code and another layer of indirection?)